src/scribe_data/extract_transform/languages/English/nouns/format_nouns.py

```diff
@@ -6,31 +6,20 @@
 """
 
 import collections
-import json
 import os
 import sys
 
+PATH_TO_SCRIBE_ORG = os.path.dirname(sys.path[0]).split("Scribe-Data")[0]
+PATH_TO_SCRIBE_DATA_SRC = f"{PATH_TO_SCRIBE_ORG}Scribe-Data/src"
+sys.path.insert(0, PATH_TO_SCRIBE_DATA_SRC)
+from scribe_data.utils import export_formatted_data, load_queried_data
```

Comment:
Could we get a space between the path definitions and the import statements? Would be a bit cleaner :)

Reply:
done